charts/ocis/templates/antivirus/deployment.yaml

```
@@ -1,6 +1,6 @@
 {{ if .Values.features.virusscan.enabled }}
 {{- include "ocis.appNames" (dict "scope" . "appName" "appNameAntivirus" "appNameSuffix" "") -}}
-{{- $_ := set . "resources" (default (default (dict) .Values.resources) .Values.services.audit.resources) -}}
+{{- $_ := set . "resources" (default (default (dict) .Values.resources) .Values.services.antivirus.resources) -}}
```

Maybe should should also modify it as seen below (untested) to make it less error-prone:

```suggestion
{{- $_ := set . "resources" (default (default (dict) .Values.resources) (index .Values.services ".appName").resources)) -}}
```